terraform/projects/prom-ec2/paas-production/extra-prometheus-scrape-configs.yml.tpl

```diff
@@ -158,3 +158,22 @@
   # Drop the temporary label
   - regex: ^__store_this__$
     action: labeldrop
+- job_name: paas_redis_metrics_for_dm
+  scheme: https
+  basic_auth:
+    username: ${dm_paas_metrics_username}
+    password: ${dm_paas_metrics_password}
+  static_configs:
+  - targets:
+    - redis.metrics.cloud.service.gov.uk
+  metrics_path: /metrics
+  scrape_interval: 300s
+  scrape_timeout: 120s
+  honor_timestamps: true
+  metric_relabel_configs:
+  # Prepend `paas_redis_` so the metrics are easier to find
+  - action: replace
+    source_labels: [__name__]
+    target_label: __name__
+    regex: (.*)
+    replacement: paas_redis_${1}
```

Comment:
Does this need a double-dollar by any chance?

Reply:
@risicle @46bit I took this directly from the documentation at https://github.com/alphagov/paas-prometheus-endpoints/tree/main/src/redis#setup as such please may the documentation be updated?

Reply:
added in 9a081d2

Reply:
Ah I think _that_ example is probably right, the extra dollar is because this is actually a `.yml.tpl` - a terraform-template of yaml, and the extra `$` is to escape the terraform templating.

Reply:
the double dollar is because of this codebase, not relevant to those other docs. sorry this is such a pain--custom config of the observe prometheus is something i added in a hurry and it's not a widely-used feature. ping me on slack if you want to pair